src/Pager.php

```diff
@@ -54,7 +54,7 @@ public function makelink()
         }
 
         $parameters[$pageid] = '';
-        $link = '?' . http_build_query($parameters);
+        $link = $this->app['paths']['current'] . '?' . http_build_query($parameters);
```

Comment:
`$app['paths']` is a hangover from 0.x days and should not be used.

See `$app['resources']->getPath()`